docs/Analyzer Configuration.md

```diff
@@ -266,6 +266,9 @@ Example: `dotnet_code_quality.try_determine_additional_string_formatting_methods
 
 Configurable Rules:
 [CA1001](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1001),
+[CA1054](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1054),
+[CA1055](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1055),
+[CA1056](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1056),
```

Comment:
Can we also create an issue on dotnet/docs repo for this change?

Reply:
PR is in progress and linked above :)